### DISCUSSION TITLE
Updates?

### DISCUSSION THREAD

**Discussion by pgale** [hugobloem/stateful_scenes#89] [General]
Just wondering if you're going to be able to do any more work on this. We were in the middle of troubleshooting the latest beta several weeks ago and I'm keen to get it working. Let me know if you aren't able to and I'll temporarily delete it until you're able to put more time to it. Many thanks - excited to see it working eventually.

**Comment by hugobloem** [comment]
Hi, I am very keen to get back to this but haven't had much time over the past weeks. I will have time this weekend to work on this!

**Reply by pgale** [reply]
Thank you. I understand! Would be great to get it working.

**Comment by pgale** [comment]
I've seen all the work you've done on b3 - I couldn't see anything specifically about hue scenes - is that still to work on or should I test b3 now?
